net_util/src/tap.rs

```diff
@@ -68,18 +70,19 @@ fn create_socket() -> Result<UdpSocket> {
 
 // Returns a byte vector representing the contents of a null terminated C string which
 // contains if_name.
-fn build_terminated_if_name(if_name: &str) -> Result<Vec<u8>> {
+fn build_terminated_if_name(if_name: &str) -> Result<[u8; IFACE_NAME_MAX_LEN]> {
     // Convert the string slice to bytes, and shadow the variable,
     // since we no longer need the &str version.
     let if_name = if_name.as_bytes();
 
-    // TODO: the 16usize limit of the if_name member from struct Tap is pretty arbitrary.
+    // TODO: the IFACE_NAME_MAX_LEN (16usize) limit of the if_name member from struct Tap is
```

Comment:
It's not really arbitrary, that's the maxsize defined in the kernel.

Can you also update the commit message to specify that the change is needed to conform with the Linux Kernel definition?

Reply:
Removed this comment as it doesn't make much sense any more. Updated the commit message.